core/src/main/scala/org/http4s/rho/Router.scala

```diff
@@ -72,15 +72,8 @@ case class CodecRouter[T <: HList, R](router: Router[T], decoder: EntityDecoder[
   override def decoding[R2 >: R](decoder2: EntityDecoder[R2])(implicit t: TypeTag[R2]): CodecRouter[T, R2] =
     CodecRouter(router, decoder orElse decoder2)
 
-  override val headers: HeaderRule = {
```

Comment:
Just swagger support. Maybe I shouldn't do it, but it was creating an unnecessary header.